src/mixins/itext_key_behavior.mixin.js

```diff
@@ -11,6 +11,7 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
     this.hiddenTextarea.setAttribute('spellcheck', 'false');
     this.hiddenTextarea.setAttribute('data-fabric-hiddentextarea', '');
     this.hiddenTextarea.setAttribute('wrap', 'off');
+    this.hiddenTextarea.setAttribute('dir', this.direction);
```

Comment:
there was a reason why i held on this initially, but i don't remember.
I think because it was changing how selection gets calculated and was incosistente between firefox and chrome

Reply:
Firefox is now getting this [direction](https://bugzilla.mozilla.org/show_bug.cgi?id=1728999) issue fixed

Reply:
Fixed indeed - verified with nightly

Reply:
https://jsfiddle.net/mistriel/4mu386t5/  works similar on both browsers !